src/bionics.cpp

```diff
@@ -2101,8 +2120,8 @@ bool Character::can_uninstall_bionic( const bionic_id &b_id, player &installer,
         }
     }
 
-    if( b_id == bio_eye_optic ) {
-        popup( _( "The Telescopic Lenses are part of %s eyes now.  Removing them would leave %s blind." ),
+    if( b_id->has_flag( flag_BIO_CANT_UNINSTAL ) ) {
+        popup( b_id->cant_uninstal_msg.c_str(),
```

Comment:
```suggestion
        popup( _( b_id->cant_uninstal_msg.c_str() ),
```

Need to localise this string:
![螢幕截圖 2020-12-03 23 07 38](https://user-images.githubusercontent.com/21075502/101046903-57e9dc80-35bc-11eb-9f30-4c08de3d5a03.png)